lib/diagnostic.js

```diff
@@ -5,22 +5,33 @@
 // and returns the minimum from that set
 // assume that all the argument are numbers
 const min = function min() {
-
+  let answer = arguments[0];
+  for (let i = 1; i < arguments.length; i++) {
+    if (answer < arguments[i]) {
+      answer = arguments[i];
+    }
+  }
+  return(answer);
```

Comment:
As written, this function will return the **largest** value in the set, not the smallest. The trick is that if any given element is smaller than the _smallest so far_ (which is really what we're storing inside `answer`) it becomes the new smallest element.